subprojects/kotlinReleaseCoroutinesTest/kotlinReleaseCoroutinesTest.gradle

```diff
@@ -17,6 +20,15 @@ repositories {
     maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
 }
 
+sourceCompatibility = 11
```

Comment:
Fixing this warning:

```
> Task :kotlinTest:compileTestKotlin
'compileTestJava' task (current target is 17) and 'compileTestKotlin' task (current target is 1.8) jvm target compatibility should be set to the same Java version.
By default will become an error since Gradle 8.0+! Read more: https://kotl.in/gradle/jvm/target-validation
Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain
```